dev-env/windows/manifests/bazel.json

```diff
@@ -5,13 +5,13 @@
   "bin": "bazel.exe",
   "architecture": {
     "64bit": {
-      "url": "https://github.com/bazelbuild/bazel/releases/download/2.1.0/bazel-2.1.0-windows-x86_64.zip",
-      "hash": "0c8492b49310f73cdfbc4df9173b5f2cd2aa547d8dfbe46b47b93f3d712e5864"
+      "url": "",
```

Comment:
I assume you want to undo this before merging?

Reply:
Not really "undo", but replace with the new values of the patched one. That's point 3 in my plan of action.

Reply:
:+1: